chunky/src/java/se/llbit/chunky/model/AnimatedQuadModel.java

```diff
@@ -0,0 +1,88 @@
+package se.llbit.chunky.model;
+
+import se.llbit.chunky.plugin.PluginApi;
+import se.llbit.chunky.renderer.scene.Scene;
+import se.llbit.chunky.resources.AnimatedTexture;
+import se.llbit.chunky.resources.Texture;
+import se.llbit.chunky.world.BlockData;
+import se.llbit.math.Quad;
+import se.llbit.math.Ray;
+import se.llbit.math.Vector3;
+import se.llbit.util.MinecraftPRNG;
+
+public abstract class AnimatedQuadModel extends QuadModel {
```

Comment:
Any texture can be animated in Minecraft so this approach won't work well in the future. Would it be sufficient to change the `getTextures()` method of `QuadModel` to take the time as input?